paasta_tools/autoscaling_lib.py

```diff
@@ -344,6 +344,16 @@ def humanize_error(error):
         return 'utilization within thresholds'
 
 
+def is_task_healthy(task):
+    """
+    Check a marathon task has healthcheck results
+    and check that they are all healthy.
+    """
+    if task.health_check_results:
+        return all([hcr.alive for hcr in task.health_check_results])
```

Comment:
This is kinda duplicating paasta_tools/bounce_lib.py:264, also paasta_tools/marathon_serviceinit.py:147

Can you combine these together in a marathon_tools.py function?